### ISSUE TITLE
[2.0] Installing plugin throws an SQL error 

### ISSUE THREAD

**Issue by FlusherDock1** [octobercms/october#5563]
I'm Installing Lovata.Shopaholic plugin, and this SQL error occures. Looks like system_plugin_history table has some column errors.

SQLSTATE[22001]: String data, right truncated: 1406 Data too long for column 'detail' at row 1 (SQL: insert into system_plugin_history (`code`, type, version, detail, `created_at`) values (Lovata.Shopaholic, comment, 1.4.0, Replace code of product sort  
  ing by popularity and rating from Shopaholic to extension plugins. Add event "shopaholic.sorting.get.list" for custom sorting of products. Add integration with "Related products for Shopaholic" and "Accessories for Shopaholic" plugins, 2021-04-16 07:54:39))

**Comment by tobias-kuendig** [comment]
I'm having this with other plugins as well. The `details` column is created as `VARCHAR(255)`, but the length of the changelog messages has not been limited to this length in the past. Probably should become a `TEXT` column to remain backwards compatible.

Quick workaround:

```sql
alter table system_plugin_history modify detail TEXT;
```

then

```bash
php artisan october:migrate
```

**Comment by daftspunk** [comment]
It might be best to remove the storage of these notes all together since they don't really serve any purpose. Here are the notes for this:

> In the current design, it doesn't make a whole lot of sense to save the changelog comments to the database. They're not used anywhere and just add bulk to the table for no reason. The plugin and therefore notes must be present in the system to perform a rollback, and therefore would be present when analysing the notes
>
> This can be restored if we do want to make non-plugin rollbacks possible, but it would require capturing the PHP rollback portion of the migration in the database as well

Fix will be delivered in v2.0.1
